src/lib.rs

```diff
@@ -36,6 +36,7 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 #![cfg_attr(docsrs, feature(doc_cfg))]
 #![allow(clippy::unreadable_literal)]
+#![feature(error_in_core)]
```

Comment:
Unless I am mistaken, the core::error::Error trait got stabilized in 1.81.0, so for older compilers using this feature requires nightly support, and newer compilers would complain about this feature not existing anymore. What am I missing?